<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Towards an open-source model for data and metadata standards</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<meta name="citation_title" content="Towards an open-source model for data and metadata standards">
<meta name="citation_abstract" content="Progress in machine learning and artificial intelligence promises to advance research and understanding across a wide range of fields and activities. In tandem, increased awareness of the importance of open data for reproducibility and scientific transparency is making inroads in fields that have not traditionally produced large publicly available datasets. Data sharing requirements from publishers and funders, as well as from other stakeholders, have also created pressure to make datasets with research and/or public interest value available through digital repositories. However, to make the best use of existing data, and facilitate the creation of useful future datasets, robust, interoperable and usable standards need to evolve and adapt over time. The open-source development model provides significant potential benefits to the process of standard creation and adaptation. In particular, data and meta-data standards can use long-standing technical and socio-technical processes that have been key to managing the development of software, and which allow incorporating broad community input into the formulation of these standards. On the other hand, open-source models carry unique risks that need to be considered. This report surveys existing open-source standards development, addressing these benefits and risks. It outlines recommendations for standards developers, funders and other stakeholders on the path to robust, interoperable and usable open-source data and metadata standards.
">
<meta name="citation_author" content="Ariel Rokem">
<meta name="citation_author" content="Vani Mandava">
<meta name="citation_author" content="Nicoleta Cristea">
<meta name="citation_author" content="Anshul Tambay">
<meta name="citation_author" content="Andrew J. Connolly">
<meta name="citation_language" content="en">
<meta name="citation_reference" content="citation_title=Hiring, managing, and retaining data scientists and research software engineers in academia: A career guidebook from ADSA and US-RSE;,citation_abstract=The importance of data, software, and computation has long been recognized in academia and is reflected in the recent rise of job opportunities for data scientists and research software engineers. Big data, for example, created a wave of novel job descriptions before the term Data Scientist (DS) was widely used. And even though software has become a major driver for research (Nangia and Katz, 2017), Research Software Engineer (RSE) as a formal role has lagged behind in terms of job openings, recognition, and prominence within the community. Despite their importance in the academic research ecosystem, the value of DS and RSE roles is not yet widely understood or appreciated in the academic community, and research data, software, and workflows are, in many domains, still regarded as by-products of research. Data Scientists and Research Software Engineers (DS/RSEs) face similar challenges when it comes to career paths in academia - both are non-traditional academic professions with few incentives and a lack of clear career trajectories. This guidebook presents the challenges and suggestions for solutions to improve the situation and to reach a wide community of stakeholders needed to advance career paths for DS/RSEs.;,citation_editor=Steve Van Tuyl;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url=https://zenodo.org/records/8329337;,citation_doi=https://doi.org/10.5281/zenodo.8329337;">
<meta name="citation_reference" content="citation_title=Information blocking: Is it occurring and what policy strategies can address it?;,citation_abstract=Policy Points: Congress has expressed concern about electronic health record (EHR) vendors and health care providers knowingly interfering with the electronic exchange of patient health informatio...;,citation_author=Julia Adler-Milstein;,citation_author=Eric Pfeifer;,citation_publication_date=2017-03;,citation_cover_date=2017-03;,citation_year=2017;,citation_issue=1;,citation_volume=95;,citation_journal_title=Milbank Q.;,citation_publisher=John Wiley &amp;amp;amp; Sons, Ltd;">
<meta name="citation_reference" content="citation_title=A national survey of digital health company experiences with electronic health record application programming interfaces;,citation_abstract=OBJECTIVES: This study sought to capture current digital health company experiences integrating with electronic health records (EHRs), given new federally regulated standards-based application programming interface (API) policies. MATERIALS AND METHODS: We developed and fielded a survey among companies that develop solutions enabling human interaction with an EHR API. The survey was developed by the University of California San Francisco in collaboration with the Office of the National Coordinator for Health Information Technology, the California Health Care Foundation, and ScaleHealth. The instrument contained questions pertaining to experiences with API integrations, barriers faced during API integrations, and API-relevant policy efforts. RESULTS: About 73% of companies reported current or previous use of a standards-based EHR API in production. About 57% of respondents indicated using both standards-based and proprietary APIs to integrate with an EHR, and 24% worked about equally with both APIs. Most companies reported use of the Fast Healthcare Interoperability Resources standard. Companies reported that standards-based APIs required on average less burden than proprietary APIs to establish and maintain. However, companies face barriers to adopting standards-based APIs, including high fees, lack of realistic clinical testing data, and lack of data elements of interest or value. DISCUSSION: The industry is moving toward the use of standardized APIs to streamline data exchange, with a majority of digital health companies using standards-based APIs to integrate with EHRs. However, barriers persist. CONCLUSION: A large portion of digital health companies use standards-based APIs to interoperate with EHRs. Continuing to improve the resources for digital health companies to find, test, connect, and use these APIs “without special effort” will be crucial to ensure future technology robustness and durability.;,citation_author=Wesley Barker;,citation_author=Natalya Maisel;,citation_author=Catherine E Strawley;,citation_author=Grace K Israelit;,citation_author=Julia Adler-Milstein;,citation_author=Benjamin Rosner;,citation_publication_date=2024-04;,citation_cover_date=2024-04;,citation_year=2024;,citation_issue=4;,citation_volume=31;,citation_journal_title=J. Am. Med. Inform. Assoc.;,citation_publisher=Oxford Academic;">
<meta name="citation_reference" content="citation_title=ODIN: Open data in neurophysiology: Advancements, solutions &amp;amp;amp; challenges;,citation_abstract=Across the life sciences, an ongoing effort over the last 50 years has made data and methods more reproducible and transparent. This openness has led to transformative insights and vastly accelerated scientific progress. For example, structural biology and genomics have undertaken systematic collection and publication of protein sequences and structures over the past half-century, and these data have led to scientific breakthroughs that were unthinkable when data collection first began. We believe that neuroscience is poised to follow the same path, and that principles of open data and open science will transform our understanding of the nervous system in ways that are impossible to predict at the moment. To this end, new social structures along with active and open scientific communities are essential to facilitate and expand the still limited adoption of open science practices in our field. Unified by shared values of openness, we set out to organize a symposium for Open Data in Neuroscience (ODIN) to strengthen our community and facilitate transformative neuroscience research at large. In this report, we share what we learned during this first ODIN event. We also lay out plans for how to grow this movement, document emerging conversations, and propose a path toward a better and more transparent science of tomorrow.;,citation_author=Colleen J Gillon;,citation_author=Cody Baker;,citation_author=Ryan Ly;,citation_author=Edoardo Balzani;,citation_author=Bingni W Brunton;,citation_author=Manuel Schottdorf;,citation_author=Satrajit Ghosh;,citation_author=Nima Dehghani;,citation_publication_date=2024-07;,citation_cover_date=2024-07;,citation_year=2024;,citation_journal_title=arXiv [q-bio.NC];">
<meta name="citation_reference" content="citation_title=How can intracranial EEG data be published in a standardized format?;,citation_abstract=Sharing data or code with publications is not something new and licenses for public sharing have existed since the late 20s century. More recent worldwide efforts have led to an increase in the amount of data shared: funding agencies require that data are shared, journals request that data are made available, and some journals publish papers describing data resources. For intracranial EEG (iEEG) data, considering how and when to share data does not happen only at the stage of publication. Human subjects’ rights demand that data sharing is something that should be considered when writing an ethical protocol and designing a study before data are collected. At that moment, it should already be considered what levels of data will be collected and potentially shared. This includes levels of data directly from the amplifier, reformatted or processed data, clinical information and imaging data. In this chapter we will describe considerations and scholarship behind sharing iEEG data, to make it easier for the iEEG community to share data for reproducibility, teaching, advancing computational efforts, integrating iEEG data with other modalities and allow others to build on previous work.;,citation_author=Dora Hermes;,citation_author=Jan Cimbalnek;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_inbook_title=Studies in neuroscience, psychology and behavioral economics;">
<meta name="citation_reference" content="citation_title=The virtual astronomical observatory: Re-engineering access to astronomical data;,citation_abstract=The US Virtual Astronomical Observatory was a software infrastructure and development project designed both to begin the establishment of an operational Virtual Observatory (VO) and to provide the US coordination with the international VO effort. The concept of the VO is to provide the means by which an astronomer is able to discover, access, and process data seamlessly, regardless of its physical location. This paper describes the origins of the VAO, including the predecessor efforts within the US National Virtual Observatory, and summarizes its main accomplishments. These accomplishments include the development of both scripting toolkits that allow scientists to incorporate VO data directly into their reduction and analysis environments and high-level science applications for data discovery, integration, analysis, and catalog cross-comparison. Working with the international community, and based on the experience from the software development, the VAO was a major contributor to international standards within the International Virtual Observatory Alliance. The VAO also demonstrated how an operational virtual observatory could be deployed, providing a robust operational environment in which VO services worldwide were routinely checked for aliveness and compliance with international standards. Finally, the VAO engaged in community outreach, developing a comprehensive web site with on-line tutorials, announcements, links to both US and internationally developed tools and services, and exhibits and hands-on training at annual meetings of the American Astronomical Society and through summer schools and community days. All digital products of the VAO Project, including software, documentation, and tutorials, are stored in a repository for community access. The enduring legacy of the VAO is an increasing expectation that new telescopes and facilities incorporate VO capabilities during the design of their data management systems.;,citation_author=R J Hanisch;,citation_author=G B Berriman;,citation_author=T J W Lazio;,citation_author=S Emery Bunn;,citation_author=J Evans;,citation_author=T A McGlynn;,citation_author=R Plante;,citation_publication_date=2015-06;,citation_cover_date=2015-06;,citation_year=2015;,citation_volume=11;,citation_journal_title=Astron. Comput.;,citation_publisher=Elsevier BV;">
<meta name="citation_reference" content="citation_title=Thirty years of the DICOM standard;,citation_abstract=Digital Imaging and Communications in Medicine (DICOM) is an international standard that defines a format for storing medical images and a protocol to enable and facilitate data communication among medical imaging systems. The DICOM standard has been instrumental in transforming the medical imaging world over the last three decades. Its adoption has been a significant experience for manufacturers, healthcare users, and research scientists. In this review, thirty years after introducing the standard, we discuss the innovation, advantages, and limitations of adopting the DICOM and its possible future directions.;,citation_author=Michele Larobina;,citation_publication_date=2023-10;,citation_cover_date=2023-10;,citation_year=2023;,citation_issue=5;,citation_volume=9;,citation_journal_title=Tomography;,citation_publisher=mdpi.com;">
<meta name="citation_reference" content="citation_title=Overview of the DICOM standard;,citation_abstract=Digital technology has in the last few decades entered almost every aspect of medicine. There has been a huge development in noninvasive medical imaging equipment. Because there are many medical equipment manufacturers, a standard for storage and exchange of medical images needed to be developed. DICOM (Digital Imaging and Communication in Medicine) makes medical image exchange more easy and independent of the imaging equipment manufacturer. Besides the image data, DICOM file format supports other information useful to describe the image. This makes DICOM easy to use and the data exchange fast and safe while avoiding possible confusion caused by multiple files for the same study.;,citation_author=Mario Mustra;,citation_author=Kresimir Delac;,citation_author=Mislav Grgic;,citation_publication_date=2008-09;,citation_cover_date=2008-09;,citation_year=2008;,citation_volume=1;,citation_conference_title=2008 50th international symposium ELMAR;,citation_conference=IEEE;">
<meta name="citation_reference" content="citation_title=Once FITS, always FITS? Astronomical infrastructure in transition;,citation_abstract=The flexible interchange transport system (FITS) file format has become the de facto standard for sharing, analyzing, and archiving astronomical data over the last four decades. FITS was adopted by astronomers in the early 1980s to overcome incompatibilities between operating systems. On the back of FITS’ success, astronomical data became both backward compatible and easily shareable. However, new advances in the astronomical instrumentation, computational technologies, and analytic techniques have resulted in new data that do not work well within the traditional FITS format. Tensions have arisen between the desire to update the format to meet new analytic challenges and adherence to the original edict for the FITS file format to be backward compatible. We examine three inflection points in the governance of FITS: first, initial development and success, second, widespread acceptance and governance by the working group, and third, the challenges to FITS in a new era of increasing data and computational complexity within astronomy.;,citation_author=Michael Scroggins;,citation_author=Bernadette M Boscoe;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=2;,citation_volume=42;,citation_journal_title=IEEE Ann. Hist. Comput.;,citation_publisher=IEEE;">
<meta name="citation_reference" content="citation_title=Without appropriate metadata, data-sharing mandates are pointless;,citation_abstract=Funders and investigators must demand appropriate metadata standards to take data from foul to FAIR. Funders and investigators must demand appropriate metadata standards to take data from foul to FAIR.;,citation_author=Mark A Musen;,citation_publication_date=2022-09;,citation_cover_date=2022-09;,citation_year=2022;,citation_issue=7926;,citation_volume=609;,citation_journal_title=Nature;,citation_publisher=Springer Science; Business Media LLC;">
<meta name="citation_reference" content="citation_title=Zarr-developers/zarr-python: v3.0.0-alpha;,citation_author=Alistair Miles;,citation_author=undefined jakirkham;,citation_author=M Bussonnier;,citation_author=Josh Moore;,citation_author=Dimitri Papadopoulos Orfanos;,citation_author=Davis Bennett;,citation_author=David Stansby;,citation_author=Joe Hamman;,citation_author=James Bourbeau;,citation_author=Andrew Fulton;,citation_author=Gregory Lee;,citation_author=Ryan Abernathey;,citation_author=Norman Rzepka;,citation_author=Zain Patel;,citation_author=Mads R. B. Kristensen;,citation_author=Sanket Verma;,citation_author=Saransh Chopra;,citation_author=Matthew Rocklin;,citation_author=AWA BRANDON AWA;,citation_author=Max Jones;,citation_author=Martin Durant;,citation_author=Elliott Sales Andrade;,citation_author=Vincent Schut;,citation_author=undefined dussin;,citation_author=Shivank Chaudhary;,citation_author=Chris Barnes;,citation_author=Juan Nunez-Iglesias;,citation_author=undefined shikharsg;,citation_publication_date=2024-06;,citation_cover_date=2024-06;,citation_year=2024;,citation_fulltext_html_url=https://doi.org/10.5281/zenodo.11592827;,citation_doi=10.5281/zenodo.11592827;,citation_publisher=Zenodo;">
<meta name="citation_reference" content="citation_title=CloudBank: Managed Services to Simplify Cloud Access for Computer Science Research and Education;,citation_abstract=CloudBank is a cloud access entity founded to enable the computer science research and education communities to harness the profound computational potential of public clouds. By delivering a set of managed services designed to alleviate common points of friction associated with cloud adoption, Cloudbank serves as an integrated service provider to the research and education community. These services include front-line help desk support, cloud solution consulting, training, account management, cost monitoring and optimization support, and automated billing. CloudBank has a multi-cloud pay-per-use billing model and aims to serve the spectrum of cloud users from novice to advanced.;,citation_author=Michael Norman;,citation_author=Vince Kellen;,citation_author=Shava Smallen;,citation_author=Brian DeMeulle;,citation_author=Shawn Strande;,citation_author=Ed Lazowska;,citation_author=Naomi Alterman;,citation_author=Rob Fatland;,citation_author=Sarah Stone;,citation_author=Amanda Tan;,citation_author=Katherine Yelick;,citation_author=Eric Van Dusen;,citation_author=James Mitchell;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_fulltext_html_url=https://doi.org/10.1145/3437359.3465586;,citation_doi=10.1145/3437359.3465586;,citation_isbn=9781450382922;,citation_conference_title=Practice and experience in advanced research computing;,citation_conference=Association for Computing Machinery;,citation_series_title=PEARC ’21;">
<meta name="citation_reference" content="citation_title=Software Engineering Practices in Academia: Promoting the 3Rs—Readability, Resilience, and Reuse;,citation_author=Andrew Connolly;,citation_author=Joseph Hellerstein;,citation_author=Naomi Alterman;,citation_author=David Beck;,citation_author=Rob Fatland;,citation_author=Ed Lazowska;,citation_author=Vani Mandava;,citation_author=Sarah Stone;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=2;,citation_volume=5;,citation_journal_title=Harvard Data Science Review;">
<meta name="citation_reference" content="citation_title=A community-driven development of the brain imaging data standard (BIDS) to describe macroscopic brain connections;,citation_author=Franco Pestilli;,citation_author=Russ Poldrack;,citation_author=Ariel Rokem;,citation_author=Theodore Satterthwaite;,citation_author=Franklin Feingold;,citation_author=Eugene Duff;,citation_author=Cyril Pernet;,citation_author=Robert Smith;,citation_author=Oscar Esteban;,citation_author=Matt Cieslak;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_journal_title=OSF;">
<meta name="citation_reference" content="citation_title=Strategy for culture change;,citation_abstract=Strategy for Culture Change;,citation_author=Brian Nosek;,citation_publisher=https://www.cos.io/blog/strategy-for-culture-change;">
<meta name="citation_reference" content="citation_title=The past, present, and future of the brain imaging data structure (BIDS);,citation_abstract=The Brain Imaging Data Structure (BIDS) is a community-driven standard for the organization of data and metadata from a growing range of neuroscience modalities. This paper is meant as a history of how the standard has developed and grown over time. We outline the principles behind the project, the mechanisms by which it has been extended, and some of the challenges being addressed as it evolves. We also discuss the lessons learned through the project, with the aim of enabling researchers in other domains to learn from the success of BIDS.;,citation_author=Russell A Poldrack;,citation_author=Christopher J Markiewicz;,citation_author=Stefan Appelhoff;,citation_author=Yoni K Ashar;,citation_author=Tibor Auer;,citation_author=Sylvain Baillet;,citation_author=Shashank Bansal;,citation_author=Leandro Beltrachini;,citation_author=Christian G Benar;,citation_author=Giacomo Bertazzoli;,citation_author=Suyash Bhogawar;,citation_author=Ross W Blair;,citation_author=Marta Bortoletto;,citation_author=Mathieu Boudreau;,citation_author=Teon L Brooks;,citation_author=Vince D Calhoun;,citation_author=Filippo Maria Castelli;,citation_author=Patricia Clement;,citation_author=Alexander L Cohen;,citation_author=Julien Cohen-Adad;,citation_author=Sasha D’Ambrosio;,citation_author=Gilles Hollander;,citation_author=Marı́a Iglesia-Vayá;,citation_author=Alejandro Vega;,citation_author=Arnaud Delorme;,citation_author=Orrin Devinsky;,citation_author=Dejan Draschkow;,citation_author=Eugene Paul Duff;,citation_author=Elizabeth DuPre;,citation_author=Eric Earl;,citation_author=Oscar Esteban;,citation_author=Franklin W Feingold;,citation_author=Guillaume Flandin;,citation_author=Anthony Galassi;,citation_author=Giuseppe Gallitto;,citation_author=Melanie Ganz;,citation_author=Rémi Gau;,citation_author=James Gholam;,citation_author=Satrajit S Ghosh;,citation_author=Alessio Giacomel;,citation_author=Ashley G Gillman;,citation_author=Padraig Gleeson;,citation_author=Alexandre Gramfort;,citation_author=Samuel Guay;,citation_author=Giacomo Guidali;,citation_author=Yaroslav O Halchenko;,citation_author=Daniel A Handwerker;,citation_author=Nell Hardcastle;,citation_author=Peer Herholz;,citation_author=Dora Hermes;,citation_author=Christopher J Honey;,citation_author=Robert B Innis;,citation_author=Horea-Ioan Ioanas;,citation_author=Andrew Jahn;,citation_author=Agah Karakuzu;,citation_author=David B Keator;,citation_author=Gregory Kiar;,citation_author=Balint Kincses;,citation_author=Angela R Laird;,citation_author=Jonathan C Lau;,citation_author=Alberto Lazari;,citation_author=Jon Haitz Legarreta;,citation_author=Adam Li;,citation_author=Xiangrui Li;,citation_author=Bradley C Love;,citation_author=Hanzhang Lu;,citation_author=Eleonora Marcantoni;,citation_author=Camille Maumet;,citation_author=Giacomo Mazzamuto;,citation_author=Steven L Meisler;,citation_author=Mark Mikkelsen;,citation_author=Henk Mutsaerts;,citation_author=Thomas E Nichols;,citation_author=Aki Nikolaidis;,citation_author=Gustav Nilsonne;,citation_author=Guiomar Niso;,citation_author=Martin Norgaard;,citation_author=Thomas W Okell;,citation_author=Robert Oostenveld;,citation_author=Eduard Ort;,citation_author=Patrick J Park;,citation_author=Mateusz Pawlik;,citation_author=Cyril R Pernet;,citation_author=Franco Pestilli;,citation_author=Jan Petr;,citation_author=Christophe Phillips;,citation_author=Jean-Baptiste Poline;,citation_author=Luca Pollonini;,citation_author=Pradeep Reddy Raamana;,citation_author=Petra Ritter;,citation_author=Gaia Rizzo;,citation_author=Kay A Robbins;,citation_author=Alexander P Rockhill;,citation_author=Christine Rogers;,citation_author=Ariel Rokem;,citation_author=Chris Rorden;,citation_author=Alexandre Routier;,citation_author=Jose Manuel Saborit-Torres;,citation_author=Taylor Salo;,citation_author=Michael Schirner;,citation_author=Robert E Smith;,citation_author=Tamas Spisak;,citation_author=Julia Sprenger;,citation_author=Nicole C Swann;,citation_author=Martin Szinte;,citation_author=Sylvain Takerkart;,citation_author=Bertrand Thirion;,citation_author=Adam G Thomas;,citation_author=Sajjad Torabian;,citation_author=Gael Varoquaux;,citation_author=Bradley Voytek;,citation_author=Julius Welzel;,citation_author=Martin Wilson;,citation_author=Tal Yarkoni;,citation_author=Krzysztof J Gorgolewski;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_journal_title=ArXiv;">
<meta name="citation_reference" content="citation_title=Data stewardship for open science: Implementing FAIR principles;,citation_author=Barend Mons;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_doi=10.1201/9781315380711;,citation_isbn=9780815348184;,citation_volume=1;">
<meta name="citation_reference" content="citation_title=Observatories of the mind;,citation_abstract=An ambitious project to map the mouse brain at the Allen Institute for Brain Science is a huge undertaking that may unify neuroscience, argue Christof Koch and R. Clay Reid.;,citation_author=Christof Koch;,citation_author=R Clay Reid;,citation_publication_date=2012-03;,citation_cover_date=2012-03;,citation_year=2012;,citation_publisher=http://dx.doi.org/10.1038/483397a;">
<meta name="citation_reference" content="citation_title=Data preservation in high energy physics;,citation_abstract=Data preservation is a mandatory specification for any present and future experimental facility and it is a cost-effective way of doing fundamental research by exploiting unique data sets in the light of the continuously increasing theoretical understanding. This document summarizes the status of data preservation in high energy physics. The paradigms and the methodological advances are discussed from a perspective of more than ten years of experience with a structured effort at international level. The status and the scientific return related to the preservation of data accumulated at large collider experiments are presented, together with an account of ongoing efforts to ensure long-term analysis capabilities for ongoing and future experiments. Transverse projects aimed at generic solutions, most of which are specifically inspired by open science and FAIR principles, are presented as well. A prospective and an action plan are also indicated.;,citation_author=T Basaglia;,citation_author=M Bellis;,citation_author=J Blomer;,citation_author=J Boyd;,citation_author=C Bozzi;,citation_author=D Britzger;,citation_author=S Campana;,citation_author=C Cartaro;,citation_author=G Chen;,citation_author=B Couturier;,citation_author=G David;,citation_author=C Diaconu;,citation_author=A Dobrin;,citation_author=D Duellmann;,citation_author=M Ebert;,citation_author=P Elmer;,citation_author=J Fernandes;,citation_author=L Fields;,citation_author=P Fokianos;,citation_author=G Ganis;,citation_author=A Geiser;,citation_author=M Gheata;,citation_author=J B Gonzalez Lopez;,citation_author=T Hara;,citation_author=L Heinrich;,citation_author=M Hildreth;,citation_author=K Herner;,citation_author=B Jayatilaka;,citation_author=M Kado;,citation_author=O Keeble;,citation_author=A Kohls;,citation_author=K Naim;,citation_author=C Lange;,citation_author=K Lassila-Perini;,citation_author=S Levonian;,citation_author=M Maggi;,citation_author=Z Marshall;,citation_author=P Mato Vila;,citation_author=A Mečionis;,citation_author=A Morris;,citation_author=S Piano;,citation_author=M Potekhin;,citation_author=M Schröder;,citation_author=U Schwickerath;,citation_author=E Sexton-Kennedy;,citation_author=T Šimko;,citation_author=T Smith;,citation_author=D South;,citation_author=A Verbytskyi;,citation_author=M Vidal;,citation_author=A Vivace;,citation_author=L Wang;,citation_author=G Watt;,citation_author=T Wenaus;,citation_author=DPHEP Collaboration;,citation_publication_date=2023-09;,citation_cover_date=2023-09;,citation_year=2023;,citation_issue=9;,citation_volume=83;,citation_journal_title=The European Physical Journal C;">
<meta name="citation_reference" content="citation_title=FITS-a flexible image transport system;,citation_author=Donald Carson Wells;,citation_author=Eric W Greisen;,citation_publication_date=1979;,citation_cover_date=1979;,citation_year=1979;,citation_conference_title=Image processing in astronomy;">
<meta name="citation_reference" content="citation_title=The neurodata without borders ecosystem for neurophysiological data science;,citation_abstract=The neurophysiology of cells and tissues are monitored electrophysiologically and optically in diverse experiments and species, ranging from flies to humans. Understanding the brain requires integration of data across this diversity, and thus these data must be findable, accessible, interoperable, and reusable (FAIR). This requires a standard language for data and metadata that can coevolve with neuroscience. We describe design and implementation principles for a language for neurophysiology data. Our open-source software (Neurodata Without Borders, NWB) defines and modularizes the interdependent, yet separable, components of a data language. We demonstrate NWB’s impact through unified description of neurophysiology data across diverse modalities and species. NWB exists in an ecosystem, which includes data management, analysis, visualization, and archive tools. Thus, the NWB data language enables reproduction, interchange, and reuse of diverse neurophysiology data. More broadly, the design principles of NWB are generally applicable to enhance discovery across biology through data FAIRness.;,citation_author=Oliver Rübel;,citation_author=Andrew Tritt;,citation_author=Ryan Ly;,citation_author=Benjamin K Dichter;,citation_author=Satrajit Ghosh;,citation_author=Lawrence Niu;,citation_author=Pamela Baker;,citation_author=Ivan Soltesz;,citation_author=Lydia Ng;,citation_author=Karel Svoboda;,citation_author=Loren Frank;,citation_author=Kristofer E Bouchard;,citation_publication_date=2022-10;,citation_cover_date=2022-10;,citation_year=2022;,citation_volume=11;,citation_journal_title=Elife;">
<meta name="citation_reference" content="citation_title=The Brain Imaging Data Structure, a format for organizing and describing outputs of neuroimaging experiments;,citation_abstract=The development of magnetic resonance imaging (MRI) techniques has defined modern neuroimaging. Since its inception, tens of thousands of studies using techniques such as functional MRI and diffusion weighted imaging have allowed for the non-invasive study of the brain. Despite the fact that MRI is routinely used to obtain data for neuroscience research, there has been no widely adopted standard for organizing and describing the data collected in an imaging experiment. This renders sharing and reusing data (within or between labs) difficult if not impossible and unnecessarily complicates the application of automatic pipelines and quality assurance protocols. To solve this problem, we have developed the Brain Imaging Data Structure (BIDS), a standard for organizing and describing MRI datasets. The BIDS standard uses file formats compatible with existing software, unifies the majority of practices already common in the field, and captures the metadata necessary for most common data processing operations.;,citation_author=Krzysztof J Gorgolewski;,citation_author=Tibor Auer;,citation_author=Vince D Calhoun;,citation_author=R Cameron Craddock;,citation_author=Samir Das;,citation_author=Eugene P Duff;,citation_author=Guillaume Flandin;,citation_author=Satrajit S Ghosh;,citation_author=Tristan Glatard;,citation_author=Yaroslav O Halchenko;,citation_author=Daniel A Handwerker;,citation_author=Michael Hanke;,citation_author=David Keator;,citation_author=Xiangrui Li;,citation_author=Zachary Michael;,citation_author=Camille Maumet;,citation_author=B Nolan Nichols;,citation_author=Thomas E Nichols;,citation_author=John Pellman;,citation_author=Jean-Baptiste Poline;,citation_author=Ariel Rokem;,citation_author=Gunnar Schaefer;,citation_author=Vanessa Sochat;,citation_author=William Triplett;,citation_author=Jessica A Turner;,citation_author=Gaël Varoquaux;,citation_author=Russell A Poldrack;,citation_publication_date=2016-06;,citation_cover_date=2016-06;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/sdata201644;,citation_volume=3;,citation_journal_title=Sci Data;">
<meta name="citation_reference" content="citation_title=The FAIR guiding principles for scientific data management and stewardship;,citation_abstract=There is an urgent need to improve the infrastructure supporting the reuse of scholarly data. A diverse set of stakeholders-representing academia, industry, funding agencies, and scholarly publishers-have come together to design and jointly endorse a concise and measureable set of principles that we refer to as the FAIR Data Principles. The intent is that these may act as a guideline for those wishing to enhance the reusability of their data holdings. Distinct from peer initiatives that focus on the human scholar, the FAIR Principles put specific emphasis on enhancing the ability of machines to automatically find and use the data, in addition to supporting its reuse by individuals. This Comment is the first formal publication of the FAIR Principles, and includes the rationale behind them, and some exemplar implementations in the community.;,citation_author=Mark D Wilkinson;,citation_author=Michel Dumontier;,citation_author=I Jsbrand Jan Aalbersberg;,citation_author=Gabrielle Appleton;,citation_author=Myles Axton;,citation_author=Arie Baak;,citation_author=Niklas Blomberg;,citation_author=Jan-Willem Boiten;,citation_author=Luiz Bonino Silva Santos;,citation_author=Philip E Bourne;,citation_author=Jildau Bouwman;,citation_author=Anthony J Brookes;,citation_author=Tim Clark;,citation_author=Mercè Crosas;,citation_author=Ingrid Dillo;,citation_author=Olivier Dumon;,citation_author=Scott Edmunds;,citation_author=Chris T Evelo;,citation_author=Richard Finkers;,citation_author=Alejandra Gonzalez-Beltran;,citation_author=Alasdair J G Gray;,citation_author=Paul Groth;,citation_author=Carole Goble;,citation_author=Jeffrey S Grethe;,citation_author=Jaap Heringa;,citation_author=Peter A C Hoen;,citation_author=Rob Hooft;,citation_author=Tobias Kuhn;,citation_author=Ruben Kok;,citation_author=Joost Kok;,citation_author=Scott J Lusher;,citation_author=Maryann E Martone;,citation_author=Albert Mons;,citation_author=Abel L Packer;,citation_author=Bengt Persson;,citation_author=Philippe Rocca-Serra;,citation_author=Marco Roos;,citation_author=Rene Schaik;,citation_author=Susanna-Assunta Sansone;,citation_author=Erik Schultes;,citation_author=Thierry Sengstag;,citation_author=Ted Slater;,citation_author=George Strawn;,citation_author=Morris A Swertz;,citation_author=Mark Thompson;,citation_author=Johan Lei;,citation_author=Erik Mulligen;,citation_author=Jan Velterop;,citation_author=Andra Waagmeester;,citation_author=Peter Wittenburg;,citation_author=Katherine Wolstencroft;,citation_author=Jun Zhao;,citation_author=Barend Mons;,citation_publication_date=2016-03;,citation_cover_date=2016-03;,citation_year=2016;,citation_volume=3;,citation_journal_title=Sci Data;">
<meta name="citation_reference" content="citation_title=Desirable characteristics of data repositories for federally funded research;,citation_author=The National Science and Technology Council;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_journal_title=Executive Office of the President of the United States, Tech. Rep;">
<meta name="citation_reference" content="citation_title=U.S. LEADERSHIP IN AI: A plan for federal engagement in developing technical standards and related tools;,citation_author=National Institute of Standards and Technology;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;">
<meta name="citation_reference" content="citation_title=Origin of BDFL;,citation_author=Guido Rossum;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;,citation_publisher=https://www.artima.com/weblogs/viewpost.jsp?thread=235725;">
<meta name="citation_reference" content="citation_title=How to build up big team science: A practical guide for large-scale collaborations;,citation_abstract=The past decade has witnessed a proliferation of big team science (BTS), endeavours where a comparatively large number of researchers pool their intellectual and/or material resources in pursuit of a common goal. Despite this burgeoning interest, there exists little guidance on how to create, manage and participate in these collaborations. In this paper, we integrate insights from a multi-disciplinary set of BTS initiatives to provide a how-to guide for BTS. We first discuss initial considerations for launching a BTS project, such as building the team, identifying leadership, governance, tools and open science approaches. We then turn to issues related to running and completing a BTS project, such as study design, ethical approvals and issues related to data collection, management and analysis. Finally, we address topics that present special challenges for BTS, including authorship decisions, collaborative writing and team decision-making.;,citation_author=Heidi A Baumgartner;,citation_author=Nicolás Alessandroni;,citation_author=Krista Byers-Heinlein;,citation_author=Michael C Frank;,citation_author=J Kiley Hamlin;,citation_author=Melanie Soderstrom;,citation_author=Jan G Voelkel;,citation_author=Robb Willer;,citation_author=Francis Yuen;,citation_author=Nicholas A Coles;,citation_publication_date=2023-06;,citation_cover_date=2023-06;,citation_year=2023;,citation_issue=6;,citation_volume=10;,citation_journal_title=R Soc Open Sci;">
<meta name="citation_reference" content="citation_title=Big science, team science, and open science for neuroscience;,citation_abstract=The Allen Institute for Brain Science is a non-profit private institution dedicated to basic brain science with an internal organization more commonly found in large physics projects-large teams generating complete, accurate and permanent resources for the mouse and human brain. It can also be viewed as an experiment in the sociology of neuroscience. We here describe some of the singular differences to more academic, PI-focused institutions.;,citation_author=Christof Koch;,citation_author=Allan Jones;,citation_publication_date=2016-11;,citation_cover_date=2016-11;,citation_year=2016;,citation_issue=3;,citation_volume=92;,citation_journal_title=Neuron;">
</head>

<body>

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Towards an open-source model for data and metadata standards</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Ariel Rokem <a href="mailto:arokem@uw.edu" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0003-0679-1985" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        University of Washington
                      </p>
                    <p class="affiliation">
                        University of Washington
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Vani Mandava <a href="https://orcid.org/0000-0003-3592-9453" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        University of Washington
                      </p>
                    <p class="affiliation">
                        University of Washington
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Nicoleta Cristea <a href="https://orcid.org/0000-0002-9091-0280" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        University of Washington
                      </p>
                    <p class="affiliation">
                        University of Washington
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Anshul Tambay <a href="https://orcid.org/0009-0004-9010-1223" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        University of Washington
                      </p>
                    <p class="affiliation">
                        University of Washington
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Andrew J. Connolly <a href="https://orcid.org/0000-0001-5576-8189" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        University of Washington
                      </p>
                    <p class="affiliation">
                        University of Washington
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (elsevier)</a></p></div></div></div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>Progress in machine learning and artificial intelligence promises to advance research and understanding across a wide range of fields and activities. In tandem, increased awareness of the importance of open data for reproducibility and scientific transparency is making inroads in fields that have not traditionally produced large publicly available datasets. Data sharing requirements from publishers and funders, as well as from other stakeholders, have also created pressure to make datasets with research and/or public interest value available through digital repositories. However, to make the best use of existing data, and facilitate the creation of useful future datasets, robust, interoperable and usable standards need to evolve and adapt over time. The open-source development model provides significant potential benefits to the process of standard creation and adaptation. In particular, data and meta-data standards can use long-standing technical and socio-technical processes that have been key to managing the development of software, and which allow incorporating broad community input into the formulation of these standards. On the other hand, open-source models carry unique risks that need to be considered. This report surveys existing open-source standards development, addressing these benefits and risks. It outlines recommendations for standards developers, funders and other stakeholders on the path to robust, interoperable and usable open-source data and metadata standards.</p>
      </div>
    </div>


    <div class="quarto-other-links-text-target">
    </div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract"><span class="header-section-number">1</span> Abstract</a></li>
  <li><a href="#sec-intro" id="toc-sec-intro" class="nav-link" data-scroll-target="#sec-intro"><span class="header-section-number">2</span> Introduction</a></li>
  <li><a href="#sec-use-cases" id="toc-sec-use-cases" class="nav-link" data-scroll-target="#sec-use-cases"><span class="header-section-number">3</span> Use cases</a>
  <ul class="collapse">
  <li><a href="#astronomy" id="toc-astronomy" class="nav-link" data-scroll-target="#astronomy"><span class="header-section-number">3.1</span> Astronomy</a></li>
  <li><a href="#high-energy-physics-hep" id="toc-high-energy-physics-hep" class="nav-link" data-scroll-target="#high-energy-physics-hep"><span class="header-section-number">3.2</span> High-energy physics (HEP)</a></li>
  <li><a href="#earth-sciences" id="toc-earth-sciences" class="nav-link" data-scroll-target="#earth-sciences"><span class="header-section-number">3.3</span> Earth sciences</a></li>
  <li><a href="#neuroscience" id="toc-neuroscience" class="nav-link" data-scroll-target="#neuroscience"><span class="header-section-number">3.4</span> Neuroscience</a></li>
  <li><a href="#community-science" id="toc-community-science" class="nav-link" data-scroll-target="#community-science"><span class="header-section-number">3.5</span> Community science</a></li>
  </ul></li>
  <li><a href="#sec-challenges" id="toc-sec-challenges" class="nav-link" data-scroll-target="#sec-challenges"><span class="header-section-number">4</span> Opportunities and risks for open-source standards</a>
  <ul class="collapse">
  <li><a href="#flexibility-vs.-stability" id="toc-flexibility-vs.-stability" class="nav-link" data-scroll-target="#flexibility-vs.-stability"><span class="header-section-number">4.1</span> Flexibility vs.&nbsp;Stability</a></li>
  <li><a href="#mismatches-between-standards-developers-and-user-communities" id="toc-mismatches-between-standards-developers-and-user-communities" class="nav-link" data-scroll-target="#mismatches-between-standards-developers-and-user-communities"><span class="header-section-number">4.2</span> Mismatches between standards developers and user communities</a></li>
  <li><a href="#cross-domain-gaps" id="toc-cross-domain-gaps" class="nav-link" data-scroll-target="#cross-domain-gaps"><span class="header-section-number">4.3</span> Cross-domain gaps</a></li>
  <li><a href="#data-instrumentation-issues" id="toc-data-instrumentation-issues" class="nav-link" data-scroll-target="#data-instrumentation-issues"><span class="header-section-number">4.4</span> Data instrumentation issues</a>
  <ul class="collapse">
  <li><a href="#harnessing-new-computing-paradigms-and-technologies" id="toc-harnessing-new-computing-paradigms-and-technologies" class="nav-link" data-scroll-target="#harnessing-new-computing-paradigms-and-technologies"><span class="header-section-number">4.4.1</span> Harnessing new computing paradigms and technologies</a></li>
  </ul></li>
  <li><a href="#unclear-pathways-for-standards-success-and-sustainability" id="toc-unclear-pathways-for-standards-success-and-sustainability" class="nav-link" data-scroll-target="#unclear-pathways-for-standards-success-and-sustainability"><span class="header-section-number">4.5</span> Unclear pathways for standards success and sustainability</a></li>
  </ul></li>
  <li><a href="#sec-cross-sector" id="toc-sec-cross-sector" class="nav-link" data-scroll-target="#sec-cross-sector"><span class="header-section-number">5</span> Cross-sector interactions</a>
  <ul class="collapse">
  <li><a href="#governmental-policy-setting" id="toc-governmental-policy-setting" class="nav-link" data-scroll-target="#governmental-policy-setting"><span class="header-section-number">5.1</span> Governmental policy-setting</a></li>
  <li><a href="#funding" id="toc-funding" class="nav-link" data-scroll-target="#funding"><span class="header-section-number">5.2</span> Funding</a></li>
  <li><a href="#industry" id="toc-industry" class="nav-link" data-scroll-target="#industry"><span class="header-section-number">5.3</span> Industry</a></li>
  </ul></li>
  <li><a href="#sec-recommendations" id="toc-sec-recommendations" class="nav-link" data-scroll-target="#sec-recommendations"><span class="header-section-number">6</span> Recommendations for open-source data and metadata standards</a>
  <ul class="collapse">
  <li><a href="#science-and-technology-communities" id="toc-science-and-technology-communities" class="nav-link" data-scroll-target="#science-and-technology-communities"><span class="header-section-number">6.1</span> Science and technology communities:</a>
  <ul class="collapse">
  <li><a href="#establish-standards-governance-based-on-oss-best-practices" id="toc-establish-standards-governance-based-on-oss-best-practices" class="nav-link" data-scroll-target="#establish-standards-governance-based-on-oss-best-practices"><span class="header-section-number">6.1.1</span> Establish standards governance based on OSS best practices</a></li>
  <li><a href="#foster-meta-standards-development" id="toc-foster-meta-standards-development" class="nav-link" data-scroll-target="#foster-meta-standards-development"><span class="header-section-number">6.1.2</span> Foster meta-standards development</a></li>
  <li><a href="#develop-standards-in-tandem-with-standards-associated-software" id="toc-develop-standards-in-tandem-with-standards-associated-software" class="nav-link" data-scroll-target="#develop-standards-in-tandem-with-standards-associated-software"><span class="header-section-number">6.1.3</span> Develop standards in tandem with standards-associated software</a></li>
  </ul></li>
  <li><a href="#policy-making-and-funding-entities" id="toc-policy-making-and-funding-entities" class="nav-link" data-scroll-target="#policy-making-and-funding-entities"><span class="header-section-number">6.2</span> Policy-making and funding entities:</a>
  <ul class="collapse">
  <li><a href="#fund-the-development-of-open-source-standards" id="toc-fund-the-development-of-open-source-standards" class="nav-link" data-scroll-target="#fund-the-development-of-open-source-standards"><span class="header-section-number">6.2.1</span> Fund the development of open-source standards</a></li>
  <li><a href="#invest-in-data-stewards" id="toc-invest-in-data-stewards" class="nav-link" data-scroll-target="#invest-in-data-stewards"><span class="header-section-number">6.2.2</span> Invest in data stewards</a></li>
  <li><a href="#review-open-source-standards-pathways" id="toc-review-open-source-standards-pathways" class="nav-link" data-scroll-target="#review-open-source-standards-pathways"><span class="header-section-number">6.2.3</span> Review open-source standards pathways</a></li>
  <li><a href="#manage-cross-sector-alliances" id="toc-manage-cross-sector-alliances" class="nav-link" data-scroll-target="#manage-cross-sector-alliances"><span class="header-section-number">6.2.4</span> Manage Cross Sector alliances</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#acknowledgements" id="toc-acknowledgements" class="nav-link" data-scroll-target="#acknowledgements"><span class="header-section-number">7</span> Acknowledgements</a></li>
  
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">9</span> References</a></li>
  </ul>
<div class="quarto-alternate-notebooks"><h2>Notebooks</h2><ul><li><a href="sections/06-acknowledgments-preview.html"><i class="bi bi-journal-code"></i>Acknowledgements</a></li><li><a href="sections/07-participants-preview.html"><i class="bi bi-journal-code"></i>Appendix: List of participants</a></li><li><a href="sections/04-cross-sector-preview.html"><i class="bi bi-journal-code"></i>Cross-sector interactions</a></li><li><a href="sections/01-introduction-preview.html"><i class="bi bi-journal-code"></i>Introduction</a></li><li><a href="sections/03-challenges-preview.html"><i class="bi bi-journal-code"></i>Opportunities and risks for open-source standards</a></li><li><a href="sections/05-recommendations-preview.html"><i class="bi bi-journal-code"></i>Recommendations for open-source data and metadata standards</a></li><li><a href="sections/02-use-cases-preview.html"><i class="bi bi-journal-code"></i>Use cases</a></li></ul></div></nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<section id="abstract" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Abstract</h1>
<p>Recent progress in machine learning and artificial intelligence promises to advance research and understanding across a wide range of fields and activities. In tandem, increased awareness of the importance of open data for reproducibility and scientific transparency is making inroads in fields that have not traditionally produced large publicly available datasets. Data sharing requirements from publishers and funders, as well as from other stakeholders, have also created pressure to make datasets with research and/or public interest value available through digital repositories. However, to make the best use of existing data, and facilitate the creation of useful future datasets, robust, interoperable and usable standards need to evolve and adapt over time. The open-source development model provides significant potential benefits to the process of standard creation and adaptation. In particular, the development and adaptation of standards can use long-standing socio-technical processes that have been key to managing the development of software, and allow incorporating broad community input into the formulation of these standards. By adhering to open-source standards to formal descriptions (e.g., by implementing schemata for standard specification, and/or by implementing automated standard validation), processes such as automated testing and continuous integration, which have been important in the development of open-source software, can be adopted in defining data and metadata standards as well. Similarly, open-source governance provides a range of stakeholders a voice in the development of standards, potentially enabling use cases and concerns that would not be taken into account in a top-down model of standards development. On the other hand, open-source models carry unique risks that need to be incorporated into the process.</p>
</section>
<section id="sec-intro" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Introduction</h1>
<p>Data-intensive discovery has become an important mode of knowledge production across many research fields and it is having a significant and broad impact across all of society. This is becoming increasingly salient as recent developments in machine learning and artificial intelligence (AI) promise to increase the value of large, multi-dimensional, heterogeneous data sources. Coupled with these new machine learning techniques, these datasets can help us understand everything from the cellular operations of the human body, through business transactions on the internet, to the structure and history of the universe. However, the development of new machine learning methods and data-intensive discovery more generally depends on Findability, Accessibility, Interoperability and Reusability (FAIR) of data <span class="citation" data-cites="Wilkinson2016FAIR">(<a href="#ref-Wilkinson2016FAIR" role="doc-biblioref">Wilkinson et al. 2016</a>)</span> as well as metadata <span class="citation" data-cites="Musen2022metadata">(<a href="#ref-Musen2022metadata" role="doc-biblioref">Musen 2022</a>)</span>. One of the main mechanisms through which the FAIR principles are promoted is the development of <em>standards</em> for data and metadata. Standards can vary in the level of detail and scope, and encompass such things as <em>file formats</em> for the storage of certain data types, <em>schemas</em> for databases that organize data, <em>ontologies</em> to describe and organize metadata in a manner that connects it to field-specific meaning, as well as mechanisms to describe <em>provenance</em> of analysis products.</p>
<p>Community-driven development of robust, adaptable and useful standards draws significant inspiration from the development of open-source software (OSS) and has many parallels and overlaps with OSS development. OSS has a long history going back to the development of the Unix operating system in the late 1960s. Over the time since its inception, the large community of developers and users of OSS have developed a host of socio-technical mechanisms that support the development and use of OSS. For example, the Open Source Initiative (OSI), a non-profit organization that was founded in the 1990s developed a set of guidelines for licensing of OSS that is designed to protect the rights of developers and users. On the technical side, tools such as the Git Source-code management system support complex and distributed open-source workflows that accelerate, streamline, and make OSS development more robust. Governance approaches have been honed to address the challenges of managing a range of stakeholder interests and to mediate between large numbers of weakly-connected individuals that contribute to OSS. When these social and technical innovations are put together they enable a host of positive defining features of OSS, such as transparency, collaboration, and decentralization. These features allow OSS to have a remarkable level of dynamism and productivity, while also retaining the ability of a variety of stakeholders to guide the evolution of the software to take their needs and interests into account.</p>
<p>Data and metadata standards that use tools and practices of OSS (“open-source standards” henceforth) reap many of the benefits that the OSS model has provided in the development of other technologies. The present report explores how OSS processes and tools have affected the development of data and metadata standards. The report will survey common features of a variety of use cases; it will identify some of the challenges and pitfalls of this mode of standards development, with a particular focus on cross-sector interactions; and it will make recommendations for future developments and policies that can help this mode of standards development thrive and reach its full potential.</p>
</section>
<section id="sec-use-cases" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Use cases</h1>
<p>To understand how OSS development practices affect the development of data and metadata standards, it is informative to demonstrate this cross-fertilization through a few use cases. As we will see in these examples, some fields, such as astronomy, high-energy physics and earth sciences have a relatively long history of shared data resources from organizations such as SDSS, CERN, and NASA, while other fields have only relatively recently become aware of the value of data sharing and its impact. These disparate histories inform how standards have evolved and how OSS practices have pervaded their development. It also demonstrates field-specific limitations on the adoption of OSS tools and practices that exemplify some of the challenges, which we will explore subsequently.</p>
<section id="astronomy" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="astronomy"><span class="header-section-number">3.1</span> Astronomy</h2>
<p>An early prominent example of a community-driven standard is the FITS (Flexible Image Transport System) file format standard, which was developed in the late 1970s and early 1980s <span class="citation" data-cites="wells1979fits">(<a href="#ref-wells1979fits" role="doc-biblioref">Wells and Greisen 1979</a>)</span>, and has been adopted worldwide for astronomy data preservation and exchange. Essentially every software platform used in astronomy reads and writes the FITS format. It was developed by observatories in the 1980s to store image data in the visible and x-ray spectrum. It has been endorsed by the International Astronomical Union (IAU), as well as funding agencies. Though the format has evolved over time, “once FITS, always FITS”. That is, the format cannot be evolved to introduce changes that break backward compatibility. Among the features that make FITS so durable is that it was designed originally to have a very restricted metadata schema. That is, FITS records were designed to be the lowest common denominator of word lengths in computer systems at the time. However, while FITS is compact, its ability to encode a coordinate frame for pixels, means that data from different observational instruments can be stored in this format and relationships between data from different instruments can be defined, rendering manual and error-prone procedures for conforming images obsolete. Nevertheless, the stability has also raised some issues as the field continues to adapt to new measurement methods and the demands of ever-increasing data volumes and complex data analysis use-case, such as interchange with other data and the use of complex data bases to store and share data <span class="citation" data-cites="Scroggins2020-ut">(<a href="#ref-Scroggins2020-ut" role="doc-biblioref">Scroggins and Boscoe 2020</a>)</span>. Another prominent example of the use of open-source processes to develop standards in Astronomy is in the tools and protocols developed by the International Virtual Observatory Alliance (IVOA) and its national implementations, e.g., in the US Virtual Astronomical Observatory<span class="citation" data-cites="Hanisch2015-cu">(<a href="#ref-Hanisch2015-cu" role="doc-biblioref">Hanisch et al. 2015</a>)</span>. The virtual observatories facilitate discovery and access across observatories around the world and underpin data discovery in astronomy. The IVOA took inspiration from the World-Wide Web Consortium (W3C) and adopted its process for the development of its standards (i.e., Working drafts <span class="math inline">\(\rightarrow\)</span> Proposed Recommendations <span class="math inline">\(\rightarrow\)</span> Recommendations), with individual standards developed by inter-institutional and international working groups. One of the outcomes of the coordination effort is the development of an ecosystem of software tools both developed within the observatory teams and within the user community that interoperate with the standards that were adopted by the observatories.</p>
</section>
<section id="high-energy-physics-hep" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="high-energy-physics-hep"><span class="header-section-number">3.2</span> High-energy physics (HEP)</h2>
<p>Because data collection is centralized, standards to collect and store HEP data have been established and the adoption of these standards in data analysis has high penetration <span class="citation" data-cites="Basaglia2023-dq">(<a href="#ref-Basaglia2023-dq" role="doc-biblioref">Basaglia et al. 2023</a>)</span>. A top-down approach is taken so that within every large collaboration, standards are enforced, and this adoption is centrally managed. Access to raw data is essentially impossible because of its large volume, and making it publicly available would be technically very difficult. Therefore, analysis tools are tuned specifically to the standards of the released data. Incentives to use the standards are provided by funders that require data management plans that specify how the data is shared (i.e., in a standards-compliant manner).</p>
</section>
<section id="earth-sciences" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="earth-sciences"><span class="header-section-number">3.3</span> Earth sciences</h2>
<p>The need for geospatial data exchange between different systems began to be recognized in the 1970s and 1980s, but proprietary formats still dominated. Coordinated standardization efforts brought the Open Geospatial Consortium (OGC) establishment in the 1990s, a critical step towards open standards for geospatial data. The 1990s have also seen the development of key standards such as the Network Common Data Form (NetCDF) developed by the University Corporation for Atmospheric Research (UCAR), and the Hierarchical Data Format (HDF), a set of file formats (HDF4, HDF5) that are widely used, particularly in climate research. The GeoTIFF format, which originated at NASA in the late 1990s, is extensively used to share image data. The following two decades, the 2000s-2020s, brought an expansion of open standards and integration with web technologies developed by OGC, as well as other standards such as the Keyhole Markup Language (KML) for displaying geographic data in Earth browsers. Formats suitable for cloud computing also emerged, such as the Cloud Optimized GeoTIFF (COG), followed by Zarr and Apache Parquet for array and tabular data, respectively. In 2006, the Open Source Geospatial Foundation (OSGeo, <a href="https://www.osgeo.org">https://www.osgeo.org</a>) was established, demonstrating the community’s commitment to the development of open-source geospatial technologies. While some standards have been developed in the industry (e.g., Keyhole Markup Language (KML) by Keyhole Inc., which Google later acquired), they later became international standards of the OGC, which now encompasses more than 450 commercial, governmental, nonprofit, and research organizations working together on the development and implementation of open standards <a href="https://www.ogc.org">https://www.ogc.org</a>.</p>
</section>
<section id="neuroscience" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="neuroscience"><span class="header-section-number">3.4</span> Neuroscience</h2>
<p>In contrast to the previously-mentioned fields, Neuroscience has traditionally been a “cottage industry”, where individual labs have generated experimental data designed to answer specific experimental questions. While this model still exists, the field has also seen the emergence of new modes of data production that focus on generating large shared datasets designed to answer many different questions, more akin to the data generated in large astronomy data collection efforts <span class="citation" data-cites="Koch2012-ve">(<a href="#ref-Koch2012-ve" role="doc-biblioref">Koch and Clay Reid 2012</a>)</span>. This change has been brought on through a combination of technical advances in data acquisition techniques, which now generate large and very high-dimensional/information-rich datasets, cultural changes, which have ushered in new norms of transparency and reproducibility, and funding initiatives that have encouraged this kind of data collection. However, because these changes are recent relative to the other cases mentioned above, standards for data and metadata in neuroscience have been prone to adopt many elements of modern OSS development. Two salient examples in neuroscience are the Neurodata Without Borders file format for neurophysiology data <span class="citation" data-cites="Rubel2022NWB">(<a href="#ref-Rubel2022NWB" role="doc-biblioref">Rübel et al. 2022</a>)</span> and the Brain Imaging Data Structure (BIDS) standard for neuroimaging data <span class="citation" data-cites="Gorgolewski2016BIDS">(<a href="#ref-Gorgolewski2016BIDS" role="doc-biblioref">Gorgolewski et al. 2016</a>)</span>. BIDS in particular owes some of its success to the adoption of OSS development mechanisms <span class="citation" data-cites="Poldrack2024BIDS">(<a href="#ref-Poldrack2024BIDS" role="doc-biblioref">Poldrack et al. 2024</a>)</span>. For example, small changes to the standard are managed through the GitHub pull request mechanism; larger changes are managed through a BIDS Enhancement Proposal (BEP) process that is directly inspired by the Python programming language community’s Python Enhancement Proposal procedure, which is used to introduce new ideas into the language. Though the BEP mechanism takes a slightly different technical approach, it tries to emulate the open-ended and community-driven aspects of Python development to accept contributions from a wide range of stakeholders and tap a broad base of expertise.</p>
</section>
<section id="community-science" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="community-science"><span class="header-section-number">3.5</span> Community science</h2>
<p>Another interesting use case for open-source standards is community/citizen science. An early example of this approach is OpenStreetMap <a href="https://www.openstreetmap.org">https://www.openstreetmap.org</a>, which allows users to contribute to the project development with code and data and freely use the maps and other related geospatial datasets. But this example is not unique. Overall, this approach has grown in the last 20 years and has been adopted in many different fields. It has many benefits for both the research field that harnesses the energy of non-scientist members of the community to engage with scientific data, as well as to the community members themselves who can draw both knowledge and pride in their participation in the scientific endeavor. It is also recognized that unique broader benefits are accrued from this mode of scientific research, through the inclusion of perspectives and data that would not otherwise be included. To make data accessible to community scientists, and to make the data collected by community scientists accessible to professional scientists, it needs to be provided in a manner that can be created and accessed without specialized instruments or specialized knowledge. Here, standards are needed to facilitate interactions between an in-group of expert researchers who generate and curate data and a broader set of out-group enthusiasts who would like to make meaningful contributions to the science. This creates a particularly stringent constraint on transparency and simplicity of standards. Creating these standards in a manner that addresses these unique constraints can benefit from OSS tools, with the caveat that some of these tools require additional expertise. For example, if the standard is developed using git/GitHub for versioning, this would require learning the complex and obscure technical aspects of these system that are far from easy to adopt, even for many professional scientists.</p>
</section>
</section>
<section id="sec-challenges" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Opportunities and risks for open-source standards</h1>
<p>While open-source standards benefit from the technical and social aspects of OSS, these tools and practices are associated with risks that need to be mitigated.</p>
<section id="flexibility-vs.-stability" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="flexibility-vs.-stability"><span class="header-section-number">4.1</span> Flexibility vs.&nbsp;Stability</h2>
<p>One of the defining characteristics of OSS is its dynamism and its rapid evolution. Because OSS can be used by anyone and, in most cases, contributions can be made by anyone, innovations flow into OSS in a bottom-up fashion from users/developers. Pathways to contribution by members of the community are often well-defined: both from the technical perspective (e.g., through a pull request on GitHub, or other similar mechanisms), as well as from the social perspective (e.g., whether contributors need to accept certain licensing conditions through a contributor licensing agreement) and the socio-technical perspective (e.g., how many people need to review a contribution, what are the timelines for a contribution to be reviewed and accepted, what are the release cycles of the software that make the contribution available to a broader community of users, etc.). Similarly, open-source standards may also find themselves addressing use cases and solutions that were not originally envisioned through bottom-up contributions of members of a research community to which the standard pertains. However, while this dynamism provides an avenue for flexibility it also presents a source of tension. This is because data and metadata standards apply to already existing datasets, and changes may affect the compliance of these existing datasets. These existing datasets may have a lifespan of decades, making continued compatibility crucial. Similarly, analysis technology stacks that are developed based on an existing version of a standard have to adapt to the introduction of new ideas and changes into a standard. Dynamic changes of this sort therefore risk causing a loss of faith in the standard by a user community, and migration away from the standard. Similarly, if a standard evolves too rapidly, users may choose to stick to an outdated version of a standard for a long time, creating strains on the community of developers and maintainers of a standard who will need to accommodate long deprecation cycles. On the other hand, in cases in which some forms of dynamic change is prohibited – as in the case of the FITS file format, which prohibits changes that break backwards-compatibility – there is also a cost associated with the stability <span class="citation" data-cites="Scroggins2020-ut">(<a href="#ref-Scroggins2020-ut" role="doc-biblioref">Scroggins and Boscoe 2020</a>)</span>: limiting adoption and combinations of new types of measurements, new analysis methods or new modes of data storage and data sharing.</p>
</section>
<section id="mismatches-between-standards-developers-and-user-communities" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="mismatches-between-standards-developers-and-user-communities"><span class="header-section-number">4.2</span> Mismatches between standards developers and user communities</h2>
<p>Open-source standards often entail an inherent gap between the core developers of the standard and the users of the standard. The former may be possess higher ability to engage with the technical details undergirding standards and their development, while the latter still have a high level of interest as members of the broader research field to which the standard pertains. This gap, in and of itself, creates friction on the path to broad adoption and best utilization of the standards. In extreme cases, the interests of researchers and standards developers may even seem at odds, as developers implement sophisticated mechanisms to automate the creation and validation of the standard or advocate for more technically advanced mechanisms for evolving the standard. These advanced capabilities offer more robust development practices and consistency in cases where the standards are complex and elaborate. They can also ease the maintenance burden of the standard. On the other hand, they may end up leaving potential experimental researchers and data providers sidelined in the development of the standard, and limiting their ability to provide feedback about the practical implications of changes to the standards. One example of this (already mentioned above in <a href="#sec-use-cases" class="quarto-xref">Section&nbsp;3</a>) is the use of git/GitHub for versioning of standards documents. This sets a high bar for participation in standards development for researchers in fields of research in which git/GitHub have not yet had significant adoption as tools of day-to-day computational practice. At the same time, it provides clarity and robustness for standards developers communities that are well-versed in these tools.</p>
<p>Another layer of potential mismatches arises when a more complex set of stakeholders needs to be considered. For example, the Group on Earth Observations (GEO) is a network that aims to coordinate decision making around satellite missions and to standardize the data that results from these missions. Because this group involves a range of different stakeholders, including individuals who more closely understand potential legal issues and researchers who are better equipped to evaluate technical and domain questions, communication is slower and hindered. As the group aims to move forward by consensus, these communication difficulties can slow down progress. This is just an example, which exemplifies the many cases in which OSS process which strives for consensus can slow progress.</p>
</section>
<section id="cross-domain-gaps" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="cross-domain-gaps"><span class="header-section-number">4.3</span> Cross-domain gaps</h2>
<p>There is much to be gained from the development of standards that apply in multiple different domains. For example, many research fields use images as data and array-based computing that is applicable to images in various research fields is at the core of many scientific computing codes. This means that practitioners within any given field should be motivated to draw on shared data standards and shared software implementations of operations that are common across fields. On the other hand, it is very hard to justify the investment in these common resources. On the one hand, data standardization investment is even more justified if the standard is generalizable beyond any specific science domain. On the other hand, while the use cases are domain sciences based, data standardization is seen as a data infrastructure and not a science investment, reducing the immediate incentives for researchers to engage with such efforts. This is exacerbated by science research funding schemes that eschew generalized cross-domain solutions, and that seek to generate tangible impact only with a specific domain.</p>
</section>
<section id="data-instrumentation-issues" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="data-instrumentation-issues"><span class="header-section-number">4.4</span> Data instrumentation issues</h2>
<p>Where there is commercial interest in the development of data analysis tools (e.g., in biomedical applications or applications were research funding can be directed towards commercial solutions) there is an incentive to create data formats and data analysis platforms that are proprietary. This may drive innovative applications of scientific measurements, but also creates sub-fields where scientific observations are generated by proprietary instrumentation, due to these commercialization or other profit-driven incentives. FTIR Spectroscopy is one such example, wherein use of Bruker instrumentation necessitates downstream analysis of the resulting measurements using proprietary binary formats necessary for the OPUS Software. Another example is the proliferation of proprietary file formats in electrophysiological measurements of brain signals <span class="citation" data-cites="Gillon2024-vu">Hermes and Cimbalnek (<a href="#ref-Hermes2023-aw" role="doc-biblioref">2023</a>)</span>. And yet another one is proprietary application programming interfaces (APIs) used in electronic health records <span class="citation" data-cites="Barker2024-ox">Adler-Milstein and Pfeifer (<a href="#ref-Adler-Milstein2017-id" role="doc-biblioref">2017</a>)</span>. In most cases, there is a lack of regulatory oversight to adhere to available standards or evolve common tools, limiting integration across different measurements. In cases where a significant amount of data is already stored in proprietary formats, or where access is limited by proprietary APIs significant data transformations may be required to get data to a state that is amenable to open-source standards. In these sub-fields there may also be a lack of incentive to set aside investment or resources to invest in establishing open-source data standards, leaving these sub-fields relatively siloed.</p>
<section id="harnessing-new-computing-paradigms-and-technologies" class="level3" data-number="4.4.1">
<h3 data-number="4.4.1" class="anchored" data-anchor-id="harnessing-new-computing-paradigms-and-technologies"><span class="header-section-number">4.4.1</span> Harnessing new computing paradigms and technologies</h3>
<p>Open-source standards development faces the challenges of adapting to new computing paradigms and technologies. Cloud computing provides a particularly stark set of opportunities and challenges. On the one hand, cloud computing offers practical solutions for many challenges of contemporary data-driven research. For example, the scalability of cloud resources addresses some of the challenges of the scale of data that is produced by instruments in many fields. The cloud also makes data access relatively straightforward, because of the ability to determine data access permissions in a granular fashion. On the other hand, cloud computing requires reinstrumenting many data formats. This is because cloud data access patterns are fundamentally different from the ones that are used in local posix-style file-systems. Suspicion of cloud computing comes in two different flavors: the first by researchers and administrators who may be wary of costs associated with cloud computing, and especially with the difficulty of predicting these costs. This can particularly affect scenarios where long-term preservation is required. Projects such as NSF’s Cloud Bank seek to mitigate some of these concerns, by providing an additional layer of transparency into cloud costs <span class="citation" data-cites="Norman2021CloudBank">(<a href="#ref-Norman2021CloudBank" role="doc-biblioref">Norman et al. 2021</a>)</span>. The other type of objection relates to the fact that cloud computing services, by their very nature, are closed ecosystems that resist portability and interoperability. Some aspects of the services are always going to remain hidden and privy only to the cloud computing service provider. In this respect, cloud computing runs afoul of some of the appealing aspects of OSS. That said, the development of “cloud native” standards can provide significant benefits in terms of the research that can be conducted. For example, NOAA plans to use cloud computing for integration across the multiple disparate datasets that it collects to build knowledge graphs that can be queried by researchers to answer questions that can only be answered through this integration. Putting all the data “in one place” should help with that. Adaptation to the cloud in terms of data standards has driven development of new file formats. A salient example is the ZARR format <span class="citation" data-cites="zarr">(<a href="#ref-zarr" role="doc-biblioref">Miles et al. 2024</a>)</span>, which supports random access into array-based datasets stored in cloud object storage, facilitating scalable and parallelized computing on these data. Indeed, data standards such as NWB (neuroscience) and OME (microscopy) now use ZARR as a backend for cloud-based storage. In other cases, file formats that were once not straightforward to use in the cloud, such as HDF5 and TIFF have been adapted to cloud use (e.g., through the cloud-optimized geoTIFF format).</p>
</section>
</section>
<section id="unclear-pathways-for-standards-success-and-sustainability" class="level2" data-number="4.5">
<h2 data-number="4.5" class="anchored" data-anchor-id="unclear-pathways-for-standards-success-and-sustainability"><span class="header-section-number">4.5</span> Unclear pathways for standards success and sustainability</h2>
<p>The development of open-source standards faces similar sustainability challenges to those faced by open-source software that is developed for research. Standards typically develop organically through sustained and persistent efforts from dedicated groups of data practitioners. These include scientists and the broader ecosystem of data curators and users. However, there is no playbook on the structure and components of a data standard, or the pathway that moves the implementation of a specific data architecture (e.g., a particular file format) to become a data standard. As a result, data standardization lacks formal avenues for success and recognition, for example through dedicated research grants (and see <a href="#sec-cross-sector" class="quarto-xref">Section&nbsp;5</a>). This hampers the long-term trajectory that is needed to inculcate a standard into the day-to-day practice of researchers.</p>
</section>
</section>
<section id="sec-cross-sector" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Cross-sector interactions</h1>
<p>The importance of standards stems not only from discussions within research fields about how research can best be conducted to take advantage of existing and growing datasets, but also arises from interactions with stakeholders in other sectors. Several different kinds of cross-sector interactions can be defined as having an important impact on the development of open-source standards.</p>
<section id="governmental-policy-setting" class="level2" data-number="5.1">
<h2 data-number="5.1" class="anchored" data-anchor-id="governmental-policy-setting"><span class="header-section-number">5.1</span> Governmental policy-setting</h2>
<p>The development of open practices in research has entailed an ongoing interaction and dialogue with various governmental bodies that set policies for research. For example, for research that is funded by the public, this entails an ongoing series of policy discussions that address the interactions between research communities and the general public. One way in which this manifests in the United States specifically is in memos issued by the directors of the White House Office of Science and Technology Policy (OSTP), James Holdren (in 2013) and Alondra Nelson (in 2022). While these memos focused primarily on making peer-reviewed publications funded by the US Federal government available to the general public, they also lay an increasingly detailed path toward the publication and general availability of the data that is collected in research that is funded by the US government. The general guidance and overall spirit of these memos dovetail with more specific policy guidance related to data and metadata standards. For example, the importance of standards was underscored in a recent report by the Subcommittee on Open Science of the National Science and Technology Council on the “Desirable characteristics of data repositories for federally funded research” <span class="citation" data-cites="nstc2022desirable">(<a href="#ref-nstc2022desirable" role="doc-biblioref">The National Science and Technology Council 2022</a>)</span>. The report explicitly called out the importance of “allow[ing] datasets and metadata to be accessed, downloaded, or exported from the repository in widely used, preferably non-proprietary, formats consistent with standards used in the disciplines the repository serves.” This highlights the need for data and metadata standards across a variety of different kinds of data. In addition, a report from the National Institute of Standards and Technology on “U.S. Leadership in AI: A Plan for Federal Engagement in Developing Technical Standards and Related Tools” emphasized that – specifically for the case of AI – “U.S. government agencies should prioritize AI standards efforts that are […] Consensus-based, […] Inclusive and accessible, […] Multi-path, […] Open and transparent, […] and [that] result in globally relevant and non-discriminatory standards…” <span class="citation" data-cites="NIST2019">(<a href="#ref-NIST2019" role="doc-biblioref">National Institute of Standards and Technology 2019</a>)</span>. The converging characteristics of standards that arise from these reports suggest that considerable thought needs to be given to how standards arise so that these goals are achieved. Importantly, open-source standards seem to well-match at least some of these characteristics.</p>
<p>The other side of policies is the implementation of these policies in practice by developers of open-source standards and by the communities to which the standards pertain. A compelling road map towards implementation and adoption of open science practices in general and open-source standards in particular is offered in a blog post authored by the Center for Open Science’s co-founder and executive director, Brian Nosek, entitled “Strategy for Culture Change” <span class="citation" data-cites="Nosek2019CultureChange">(<a href="#ref-Nosek2019CultureChange" role="doc-biblioref">Nosek, n.d.</a>)</span>. The core idea is that affecting a turn toward open science requires an alignment of not only incentives and values, but also technical infrastructure and user experience. A sociotechnical bridge between these pieces, which makes the adoption of standards possible, and maybe even easy, and the policy goals, arises from a community of practice that makes the adoption of standards <em>normative</em>. Once all of these pieces are in place, making adoption of open science standards <em>required</em> through policy becomes more straightforward and less onerous.</p>
</section>
<section id="funding" class="level2" data-number="5.2">
<h2 data-number="5.2" class="anchored" data-anchor-id="funding"><span class="header-section-number">5.2</span> Funding</h2>
<p>Government-set policy intersects with funding considerations. This is because it is primarily directed towards research that is funded through governmental funding agencies. For example, high-level policy guidance boils to practice in guidance for data management plans that are part of funded research. In response to the policy guidance, these have become increasingly more detailed and, for example, NSF- and NIH-funded researchers are now required to both formulate their plans with more clarity and increasingly also to share data using specified standards as a condition for funding.</p>
<p>However, there are other ways in which funding relates to the development of open-source standards. For example, through the BRAIN Initiative, the National Institutes of Health have provided key funding for the development of the Brain Imaging Data Structure standard in neuroscience. Where large governmental funding agencies may not have the resources or agility required to fund nascent or unconventional ways of formulating standards, funding by non-governmental philanthropies and other organizations can provide alternatives. One example (out of many) is the Chan-Zuckerberg Initiative program for Essential Open Source Software, which funds foundational open-source software projects that have an application in biomedical sciences. Distinct from NIH funding, however, some of this investment focuses on the development of OSS practices. For example, funding to the Arrow project that focuses on developing open-source software maintenance skills and practices, rather than a specific biomedical application.</p>
</section>
<section id="industry" class="level2" data-number="5.3">
<h2 data-number="5.3" class="anchored" data-anchor-id="industry"><span class="header-section-number">5.3</span> Industry</h2>
<p>Interactions of data and meta-data standards with commercial interests may provide specific sources of friction. This is because proprietary/closed formats of data can create difficulty at various transition points: from one instrument vendor to another, from data producer to downstream recipient/user, etc. On the other hand, in some cases, cross-sector collaborations with commercial entities may pave the way to robust and useful standards. For example, imaging measurements in human subjects (e.g., in brain imaging experiments) significantly interact with standards for medical imaging, and chiefly the Digital Imaging and Communications in Medicine (DICOM) standard, which is widely used in a range of medical imaging applications, including in clinical settings <span class="citation" data-cites="Larobina2023-vq">Mustra, Delac, and Grgic (<a href="#ref-Mustra2008-xk" role="doc-biblioref">2008</a>)</span>. The standard emerged from the demands of the clinical practice in the 1980s, as digital technologies were came into widespread use in medical imaging, through joint work of industry organizations: the American College of Radiology and the National Association of Electronic Manufacturers. One of the defining features of the DICOM standard is that it allows manufacturers of instruments to define “private fields” that are compliant with the standard, but which may include idiosyncratically organized data and/or metadata. This provides significant flexibility, but can also easily lead to the loss of important information. Nevertheless, the human brain imaging case is exemplary of a case in which industry standards and research standards coexist and need to communicate with each other effectively to advance research use-cases, while keeping up with the rapid development of the technologies.</p>
</section>
</section>
<section id="sec-recommendations" class="level1" data-number="6">
<h1 data-number="6"><span class="header-section-number">6</span> Recommendations for open-source data and metadata standards</h1>
<p>In conclusion of this report, we would like to propose a set of recommendations that distill the lessons learned from an examination of data and metadata standards through the lense of open-source software development practices. We divide this section into two parts: one aimed at the science and technology communities that develop and maintain open-source standards, and the other aimed at policy-making and funding agencies, who have an interest in fostering more efficient, more robust, and more transparent open-source standards.</p>
<section id="science-and-technology-communities" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="science-and-technology-communities"><span class="header-section-number">6.1</span> Science and technology communities:</h2>
<section id="establish-standards-governance-based-on-oss-best-practices" class="level3" data-number="6.1.1">
<h3 data-number="6.1.1" class="anchored" data-anchor-id="establish-standards-governance-based-on-oss-best-practices"><span class="header-section-number">6.1.1</span> Establish standards governance based on OSS best practices</h3>
<p>While best-practice governance principles are also relatively new in OSS communities, there is already a substantial set of prior art in this domain, on which the developers and maintainers of open-source data and metadata standards can rely. For example, it is now clear that governance principles and rules can mitigate some of the risks and challenges mentioned in <a href="#sec-challenges" class="quarto-xref">Section&nbsp;4</a>, especially for communities beyond a certain size that need to converge toward a new standard or rely on an existing standard. Developers and maintainers should review existing governance practices such as <a href="https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html#_project_and_community_governance">The Open Source Way</a>.</p>
</section>
<section id="foster-meta-standards-development" class="level3" data-number="6.1.2">
<h3 data-number="6.1.2" class="anchored" data-anchor-id="foster-meta-standards-development"><span class="header-section-number">6.1.2</span> Foster meta-standards development</h3>
<p>One of the main conclusions that arise from our survey of the landscape of existing standards is that there is significant knowledge that exists across fields and domains and that informs the development of standards within each field, but that could be surfaced to the level where it may be adopted more widely in different domains and be more broadly useful. One approach to this is a comparative approach: in this approach, a readiness and/or maturity model can be developed that assesses the challenges and opportunities that a specific standard faces at its current phase of development. Developing such a maturity model, while it goes beyond the scope of the current report, could lead to the eventual development of a meta-standard or a standard-of-standards. This would facilitate a succinct description of cross-cutting best-practices that can be used as a basis for the analysis or assessment of an existing standard, or as guidelines to develop new standards. For instance, specific barriers to adopting a data standard that take into account the size of the community and its specific technological capabilities should be considered.</p>
<p>More generally, meta-standards could include formalization for versioning of standards and interactions with specific related software. This includes amplifying formalization/guidelines on how to create standards (for example, metadata schema specifications using LinkML (https://linkml.io)). However, aspects of communication with potential user audiences (e.g., researchers in particular domains) should be taken into account as well. For example, in the quality of onboarding documentation and tools for ingestion or conversion into standards-compliant datasets.</p>
<p>An ontology for the standards-development process – for example top-down vs bottom-up, minimum number of datasets, target community size and technical expertise typical of this community, and so forth – could help guide the standards-development process towards more effective adoption and use. A set of meta-standards and high-level descriptions of the standards-development process – some of which is laid out in this report – could help standard developers avoid known pitfalls, such as the dreaded proliferation of standards, or complexity-impeded adoption. Surveying and documenting the success and failures of current standards for a specific dataset / domain can help disseminate knowledge about the standardization process. Resources such as <a href="https://fairsharing.org/">Fairsharing</a> or <a href="https://www.dcc.ac.uk/guidance/standards">Digital Curation Center</a> can help guide this process.</p>
</section>
<section id="develop-standards-in-tandem-with-standards-associated-software" class="level3" data-number="6.1.3">
<h3 data-number="6.1.3" class="anchored" data-anchor-id="develop-standards-in-tandem-with-standards-associated-software"><span class="header-section-number">6.1.3</span> Develop standards in tandem with standards-associated software</h3>
<p>Development of standards should be coupled and tightly linked with development of associated software. This produces a virtuous cycle where the use-cases and technical issues that arise in software development informs the development of the standard and vice versa. One of the lessons learned across a variety of different standards is the importance of automated validation of the standard. Automated validation is broadly seen as a requirement for the adoption of a standard and a factor in managing change of the standard over time. To advance this virtuous cycle, we recommend to make data standards machine readable, and make software creation an integral part of establishing a standard’s schema. Additionally, standards evolution should maintain software compatibility, and ability to translate and migrate between standards.</p>
</section>
</section>
<section id="policy-making-and-funding-entities" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="policy-making-and-funding-entities"><span class="header-section-number">6.2</span> Policy-making and funding entities:</h2>
<section id="fund-the-development-of-open-source-standards" class="level3" data-number="6.2.1">
<h3 data-number="6.2.1" class="anchored" data-anchor-id="fund-the-development-of-open-source-standards"><span class="header-section-number">6.2.1</span> Fund the development of open-source standards</h3>
<p>While some funding agencies already support standards development as part of the development of informatics infrastructures, data standards development should be seen as integral to science innovation and earmarked for funding in research grants, not only in specialized contexts. Funding models should encourage the development and adoption of standards, and fund associated community efforts and tools for this. The OSS model is seen as a particularly promising avenue for an investment of resources, because it builds on previously-developed procedures and technical infrastructure and because it provides avenues for the democratization of development processes and for community input along the way. At the same time, there are significant challenges associated with incentives to engage, ranging from the dilution of credit to individual contributors, and ranging through the burnout of maintainers and developers. The clarity offered by procedures for enhancement proposals and semantic versioning schemes adopted in standards development offers avenues for a range of stakeholders to propose well-defined contributions to large and field-wide standards efforts (e.g., <span class="citation" data-cites="pestilli2021community">(<a href="#ref-pestilli2021community" role="doc-biblioref">Pestilli et al. 2021</a>)</span>), and potentially helps alleviate some of these concerns by providing avenues for individual contributions to surface, as well as clarity of process, which can alleviate the risks of maintainer burnout.</p>
</section>
<section id="invest-in-data-stewards" class="level3" data-number="6.2.2">
<h3 data-number="6.2.2" class="anchored" data-anchor-id="invest-in-data-stewards"><span class="header-section-number">6.2.2</span> Invest in data stewards</h3>
<p>Advancing the development and adoption of open-source standards requires the dissemination of knowledge to researchers in a variety of fields, but this dissemination itself may not be enough without the fostering of specialized expertise. Therefore, it is important to recognize the distinct role that <em>data stewards</em> play in contemporary research. As policy demands for openness become increasingly high, it is crucial to truly support experts whose role will be to develop, maintain, and facilitate the adoption and use of open-source standards. This support needs to manifest in all stages of the career of these individuals: it will be necessary to set up programs for training for data stewards, and to invest in the career paths of individuals that receive such training so that this crucial role is encouraged. Initial proposals for the curriculum and scope of the role have already been proposed (e.g., in <span class="citation" data-cites="Mons2018DataStewardshipBook">(<a href="#ref-Mons2018DataStewardshipBook" role="doc-biblioref">Mons 2018</a>)</span>), but we identify here also a need to connect these individuals directly to the practices that exemplify open-source standards. Thus, it will be important for these individuals to be conversant in the methodology of OSS. This does not mean that they need to become software engineers – though for some of them there may be some overlap with the role of research software engineers <span class="citation" data-cites="Connolly2023Software">(<a href="#ref-Connolly2023Software" role="doc-biblioref">Connolly et al. 2023</a>)</span> – but rather that they need to become familiar with those parts of the OSS development life-cycle that are specifically useful for the development of open-source standards. For example, tools for version control, tools for versioning, and tools for creation and validation of compliant data and metadata. Stakeholder organizations should invest in training grants to establish curriculum for data and metadata standards education.</p>
<p>Ultimately, efficient use of data stewards and their knowledge will have to be applied. It is evident that not every project and every lab that produces data requires a full-time data steward. Instead, data stewardship could be centralized within organizations such as libraries, data science, or software engineering cores of larger research organizations. This would be akin to recent models for research software engineering that are becoming common in many research organization <span class="citation" data-cites="Van-Tuyl2023-vp">(<a href="#ref-Van-Tuyl2023-vp" role="doc-biblioref">Van Tuyl 2023</a>)</span>. Efficiency considerations also suggest that the development of data standards would not have its intended purpose unless funds are also allocated to the implementation of the standard in practice. Mandating standards without appropriate funding for their implementation by data producers and data users could risk hampering science and could leading to researchers doing the bare minimum to make their data “open”.</p>
</section>
<section id="review-open-source-standards-pathways" class="level3" data-number="6.2.3">
<h3 data-number="6.2.3" class="anchored" data-anchor-id="review-open-source-standards-pathways"><span class="header-section-number">6.2.3</span> Review open-source standards pathways</h3>
<p>Invest in programs that examine retrospective pathways for establishing data standards. Encourage publication of lifecycles for successful data standards. These lifecycles should include the process, creators, affiliations, grants, and adoption journeys of open-source standards. To encourage sustainable development of open-source standards, and to build on prior experience, the documentation and dissemination of lifecycles should be seen as an integral step of the work of standards creators and granting agencies. In the meanwhile, it would be good to also retroactively document the lifecycle of existing standards that are seen as success stories, and to foster the awareness of these standards. In addition, fostering research projects on the principles that underlie successful open-source standards development will help formulate new standards and iterate on existing ones. In accordance, data management plans should promote the sharing of not only data, but also metadata and descriptions of how to use it.</p>
</section>
<section id="manage-cross-sector-alliances" class="level3" data-number="6.2.4">
<h3 data-number="6.2.4" class="anchored" data-anchor-id="manage-cross-sector-alliances"><span class="header-section-number">6.2.4</span> Manage Cross Sector alliances</h3>
<p>Encourage cross-sector and cross-domain alliances that can impact successful standards creation. Invest in robust program management of these alliances to align pace and create incentives (for instance via Open Source Program Offices at Universities or other research organizations). Similar to program officers at funding agencies, standards evolution need sustained PM efforts. Multi-party partnerships should include strategic initiatives for standard establishment such as the Pistoia Alliance (<a href="https://www.pistoiaalliance.org/">https://www.pistoiaalliance.org/</a>).</p>
</section>
</section>
</section>
<section id="acknowledgements" class="level1" data-number="7">
<h1 data-number="7"><span class="header-section-number">7</span> Acknowledgements</h1>
<p>This report was produced following a <a href="https://uwescience.github.io/2024-open-source-standards-workshop/">workshop held at NSF headquarters in Alexandria, VA on April 8th-9th, 2024</a>. We would like to thank the speakers and participants in this workshop for the time and thought that they put into the workshop. A list of workshop participants is provided as an appendix.</p>
<p>The workshop and this report were funded through <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2334483&amp;HistoricalAwards=false">NSF grant #2334483</a> from the NSF <a href="https://new.nsf.gov/funding/opportunities/pathways-enable-open-source-ecosystems-pose">Pathways to Enable Open-Source Ecosystems (POSE)</a> program. The opinions expressed in this report do not necessarily reflect those of the National Science Foundation.</p>
</section>

<section id="references" class="level1 unnumbered" data-number="9">


</section>

<div id="quarto-appendix" class="default"><section id="appendix-list-of-participants" class="level1 appendix" data-number="8"><h2 class="anchored quarto-appendix-heading"><span class="header-section-number">8</span> Appendix: List of participants</h2><div class="quarto-appendix-contents">

<table class="caption-top table">
<thead>
<tr class="header">
<th>Name</th>
<th>Affiliation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Alex D Wade</td>
<td>Digital Science</td>
</tr>
<tr class="even">
<td>Alexander Szalay</td>
<td>Johns Hopkins University</td>
</tr>
<tr class="odd">
<td>Andrew Connolly</td>
<td>University of Washington</td>
</tr>
<tr class="even">
<td>Anshul Tushar Tambay</td>
<td>University of Washington</td>
</tr>
<tr class="odd">
<td>Ariel Rokem</td>
<td>University of Washington</td>
</tr>
<tr class="even">
<td>Carolina Lorena Berys</td>
<td>University of California, San Diego</td>
</tr>
<tr class="odd">
<td>Christine Kirkpatrick</td>
<td>San Diego Supercomputer Center</td>
</tr>
<tr class="even">
<td>Fernando Seabra Chirigati</td>
<td>Nature Computational Science</td>
</tr>
<tr class="odd">
<td>Jessica Morgan</td>
<td>NOAA</td>
</tr>
<tr class="even">
<td>John Relph</td>
<td>NOAA</td>
</tr>
<tr class="odd">
<td>Julia Ferraioli</td>
<td>Open Source Stories</td>
</tr>
<tr class="even">
<td>Jurriaan Hein Spaaks</td>
<td>formerly Netherlands eScience Center</td>
</tr>
<tr class="odd">
<td>Justin (Jay) Hnilo</td>
<td>Department of Energy</td>
</tr>
<tr class="even">
<td>Kalynn Elisabeth Kennon</td>
<td>Infectious Diseases Data Observatory</td>
</tr>
<tr class="odd">
<td>Kevin Christopher Booth</td>
<td>Radiant Earth</td>
</tr>
<tr class="even">
<td>Kristofer E. Bouchard</td>
<td>Lawrence Berkeley National Labs</td>
</tr>
<tr class="odd">
<td>Lea A. Shanley</td>
<td>University of California, Berkeley</td>
</tr>
<tr class="even">
<td>Michael Spannowsky</td>
<td>Durham University</td>
</tr>
<tr class="odd">
<td>Nicoleta C Cristea</td>
<td>University of Washington</td>
</tr>
<tr class="even">
<td>Nina Amla</td>
<td>NSF</td>
</tr>
<tr class="odd">
<td>Oliver Ruebel</td>
<td>Lawrence Berkeley National Labs</td>
</tr>
<tr class="even">
<td>Ray E. Habermann</td>
<td>Metadata Game Changers</td>
</tr>
<tr class="odd">
<td>Raymond (Ray) Plante</td>
<td>NIST</td>
</tr>
<tr class="even">
<td>Robert Hanisch</td>
<td>NIST</td>
</tr>
<tr class="odd">
<td>Saskia de Vries</td>
<td>Allen Institute for AI</td>
</tr>
<tr class="even">
<td>Steven Crawford</td>
<td>NASA</td>
</tr>
<tr class="odd">
<td>Vani Mandava</td>
<td>University of Washington</td>
</tr>
<tr class="even">
<td>Yaroslav Halchenko</td>
<td>Dartmouth University</td>
</tr>
<tr class="odd">
<td>Ziheng Sun</td>
<td>George Mason University</td>
</tr>
</tbody>
</table>
</div></section><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">9 References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-Adler-Milstein2017-id" class="csl-entry" role="listitem">
Adler-Milstein, Julia, and Eric Pfeifer. 2017. <span>“Information Blocking: Is It Occurring and What Policy Strategies Can Address It?”</span> <em>Milbank Q.</em> 95 (1): 117–35.
</div>
<div id="ref-Barker2024-ox" class="csl-entry" role="listitem">
Barker, Wesley, Natalya Maisel, Catherine E Strawley, Grace K Israelit, Julia Adler-Milstein, and Benjamin Rosner. 2024. <span>“A National Survey of Digital Health Company Experiences with Electronic Health Record Application Programming Interfaces.”</span> <em>J. Am. Med. Inform. Assoc.</em> 31 (4): 866–74.
</div>
<div id="ref-Basaglia2023-dq" class="csl-entry" role="listitem">
Basaglia, T, M Bellis, J Blomer, J Boyd, C Bozzi, D Britzger, S Campana, et al. 2023. <span>“Data Preservation in High Energy Physics.”</span> <em>The European Physical Journal C</em> 83 (9): 795.
</div>
<div id="ref-Connolly2023Software" class="csl-entry" role="listitem">
Connolly, Andrew, Joseph Hellerstein, Naomi Alterman, David Beck, Rob Fatland, Ed Lazowska, Vani Mandava, and Sarah Stone. 2023. <span>“<span>Software</span> <span>Engineering</span> <span>Practices</span> in <span>Academia</span>: Promoting the 3Rs—<span>Readability</span>, <span>Resilience</span>, and <span>Reuse</span>.”</span> <em>Harvard Data Science Review</em> 5 (2).
</div>
<div id="ref-Gillon2024-vu" class="csl-entry" role="listitem">
Gillon, Colleen J, Cody Baker, Ryan Ly, Edoardo Balzani, Bingni W Brunton, Manuel Schottdorf, Satrajit Ghosh, and Nima Dehghani. 2024. <span>“<span>ODIN</span>: Open Data in Neurophysiology: Advancements, Solutions &amp; Challenges.”</span> <em>arXiv [q-Bio.NC]</em>, July.
</div>
<div id="ref-Gorgolewski2016BIDS" class="csl-entry" role="listitem">
Gorgolewski, Krzysztof J, Tibor Auer, Vince D Calhoun, R Cameron Craddock, Samir Das, Eugene P Duff, Guillaume Flandin, et al. 2016. <span>“The <span>Brain</span> <span>Imaging</span> <span>Data</span> <span>Structure</span>, a Format for Organizing and Describing Outputs of Neuroimaging Experiments.”</span> <em>Sci Data</em> 3 (June): 160044. <a href="https://www.nature.com/articles/sdata201644">https://www.nature.com/articles/sdata201644</a>.
</div>
<div id="ref-Hanisch2015-cu" class="csl-entry" role="listitem">
Hanisch, R J, G B Berriman, T J W Lazio, S Emery Bunn, J Evans, T A McGlynn, and R Plante. 2015. <span>“The Virtual Astronomical Observatory: Re-Engineering Access to Astronomical Data.”</span> <em>Astron. Comput.</em> 11 (June): 190–209.
</div>
<div id="ref-Hermes2023-aw" class="csl-entry" role="listitem">
Hermes, Dora, and Jan Cimbalnek. 2023. <span>“How Can Intracranial <span>EEG</span> Data Be Published in a Standardized Format?”</span> In <em>Studies in Neuroscience, Psychology and Behavioral Economics</em>, 595–604. Cham: Springer International Publishing.
</div>
<div id="ref-Koch2012-ve" class="csl-entry" role="listitem">
Koch, Christof, and R Clay Reid. 2012. <span>“Observatories of the Mind.”</span> <a href="http://dx.doi.org/10.1038/483397a" class="uri">http://dx.doi.org/10.1038/483397a</a>.
</div>
<div id="ref-Larobina2023-vq" class="csl-entry" role="listitem">
Larobina, Michele. 2023. <span>“Thirty Years of the <span>DICOM</span> Standard.”</span> <em>Tomography</em> 9 (5): 1829–38.
</div>
<div id="ref-zarr" class="csl-entry" role="listitem">
Miles, Alistair, jakirkham, M Bussonnier, Josh Moore, Dimitri Papadopoulos Orfanos, Davis Bennett, David Stansby, et al. 2024. <span>“Zarr-Developers/Zarr-Python: V3.0.0-Alpha.”</span> Zenodo. <a href="https://doi.org/10.5281/zenodo.11592827">https://doi.org/10.5281/zenodo.11592827</a>.
</div>
<div id="ref-Mons2018DataStewardshipBook" class="csl-entry" role="listitem">
Mons, Barend. 2018. <em>Data Stewardship for Open Science: Implementing FAIR Principles</em>. 1st ed. Vol. 1. Milton: CRC Press. <a href="https://doi.org/10.1201/9781315380711">https://doi.org/10.1201/9781315380711</a>.
</div>
<div id="ref-Musen2022metadata" class="csl-entry" role="listitem">
Musen, Mark A. 2022. <span>“Without Appropriate Metadata, Data-Sharing Mandates Are Pointless.”</span> <em>Nature</em> 609 (7926): 222.
</div>
<div id="ref-Mustra2008-xk" class="csl-entry" role="listitem">
Mustra, Mario, Kresimir Delac, and Mislav Grgic. 2008. <span>“Overview of the <span>DICOM</span> Standard.”</span> In <em>2008 50th International Symposium ELMAR</em>, 1:39–44. IEEE.
</div>
<div id="ref-NIST2019" class="csl-entry" role="listitem">
National Institute of Standards and Technology. 2019. <span>“<span>U.S</span>. <span>LEADERSHIP</span> <span>IN</span> <span>AI</span>: A Plan for Federal Engagement in Developing Technical Standards and Related Tools.”</span>
</div>
<div id="ref-Norman2021CloudBank" class="csl-entry" role="listitem">
Norman, Michael, Vince Kellen, Shava Smallen, Brian DeMeulle, Shawn Strande, Ed Lazowska, Naomi Alterman, et al. 2021. <span>“<span class="nocase">CloudBank: Managed Services to Simplify Cloud Access for Computer Science Research and Education</span>.”</span> In <em>Practice and Experience in Advanced Research Computing</em>. PEARC ’21. New York, NY, USA: Association for Computing Machinery. <a href="https://doi.org/10.1145/3437359.3465586">https://doi.org/10.1145/3437359.3465586</a>.
</div>
<div id="ref-Nosek2019CultureChange" class="csl-entry" role="listitem">
Nosek, Brian. n.d. <span>“Strategy for Culture Change.”</span> <a href="https://www.cos.io/blog/strategy-for-culture-change" class="uri">https://www.cos.io/blog/strategy-for-culture-change</a>.
</div>
<div id="ref-pestilli2021community" class="csl-entry" role="listitem">
Pestilli, Franco, Russ Poldrack, Ariel Rokem, Theodore Satterthwaite, Franklin Feingold, Eugene Duff, Cyril Pernet, Robert Smith, Oscar Esteban, and Matt Cieslak. 2021. <span>“A Community-Driven Development of the Brain Imaging Data Standard (BIDS) to Describe Macroscopic Brain Connections.”</span> <em>OSF</em>.
</div>
<div id="ref-Poldrack2024BIDS" class="csl-entry" role="listitem">
Poldrack, Russell A, Christopher J Markiewicz, Stefan Appelhoff, Yoni K Ashar, Tibor Auer, Sylvain Baillet, Shashank Bansal, et al. 2024. <span>“The Past, Present, and Future of the Brain Imaging Data Structure (<span>BIDS</span>).”</span> <em>ArXiv</em>, January.
</div>
<div id="ref-Rubel2022NWB" class="csl-entry" role="listitem">
Rübel, Oliver, Andrew Tritt, Ryan Ly, Benjamin K Dichter, Satrajit Ghosh, Lawrence Niu, Pamela Baker, et al. 2022. <span>“The Neurodata Without Borders Ecosystem for Neurophysiological Data Science.”</span> <em>Elife</em> 11 (October).
</div>
<div id="ref-Scroggins2020-ut" class="csl-entry" role="listitem">
Scroggins, Michael, and Bernadette M Boscoe. 2020. <span>“Once <span>FITS</span>, Always <span>FITS</span>? Astronomical Infrastructure in Transition.”</span> <em>IEEE Ann. Hist. Comput.</em> 42 (2): 42–54.
</div>
<div id="ref-nstc2022desirable" class="csl-entry" role="listitem">
The National Science and Technology Council. 2022. <span>“Desirable Characteristics of Data Repositories for Federally Funded Research.”</span> <em>Executive Office of the President of the United States, Tech. Rep</em>.
</div>
<div id="ref-Van-Tuyl2023-vp" class="csl-entry" role="listitem">
Van Tuyl, Steve, ed. 2023. <span>“Hiring, Managing, and Retaining Data Scientists and Research Software Engineers in Academia: A Career Guidebook from <span>ADSA</span> and <span>US</span>-<span>RSE</span>.”</span> https://doi.org/<a href="https://doi.org/10.5281/zenodo.8329337">https://doi.org/10.5281/zenodo.8329337</a>.
</div>
<div id="ref-wells1979fits" class="csl-entry" role="listitem">
Wells, Donald Carson, and Eric W Greisen. 1979. <span>“FITS-a Flexible Image Transport System.”</span> In <em>Image Processing in Astronomy</em>, 445.
</div>
<div id="ref-Wilkinson2016FAIR" class="csl-entry" role="listitem">
Wilkinson, Mark D, Michel Dumontier, I Jsbrand Jan Aalbersberg, Gabrielle Appleton, Myles Axton, Arie Baak, Niklas Blomberg, et al. 2016. <span>“The <span>FAIR</span> Guiding Principles for Scientific Data Management and Stewardship.”</span> <em>Sci Data</em> 3 (March): 160018.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>